app/views/about/article-guides.directive.js

```diff
@@ -0,0 +1,128 @@
+﻿define(['app',"text!./article-guides.directive.html", 'services/articles-service'],
+function (app, template) {
+
+app.directive("articleGuides", [ function () {
+
+	return {
+		restrict   : "E",
+		template: template,
+		replace    : true,
+        transclude : false,
+        scope: {
+            type  : '@',
+            locale: '@',
+            title:  '@',
+            showFaqs:  '@'
+        },
+		controller : ["$scope", "articlesService",
+        function($scope,  articlesService)
+		{
+          
+          loadArticles();
+          $scope.introABSCH=[];
+          $scope.requirements=[];
+          $scope.help=[];
+          $scope.intro=[];
+          $scope.signIn=[];
+          $scope.dashboard=[];
+          $scope.format=[];
+          $scope.review=[];
+          $scope.publish=[];
+          $scope.expand=false;
+
+          //---------------------------------------------------------------------
+          function loadArticles(str){
+            var ag = [];
+            
+            ag.push({"$match":{"adminTags.title.en":{"$in":[encodeURIComponent("ABSCH-Guide"), "ABSCH-Guide-"  + $scope.type.toUpperCase() ]}}});
+            ag.push({"$project" : {"title":1, "summary":1,"content":1,"meta":1, "adminTags":1}});
+            
+            var qs = {
+              "ag" : JSON.stringify(ag)
+              
+            };
+            articlesService.getArticles(qs).then(function(data){
+              $scope.articles = data;
+            
+              for(var i=0;i < data.length;++i){
+
+                  var admintags = JSON.stringify(data[i].adminTags);
+                  console.log("a="+admintags);
+
+                  if(admintags.indexOf("ABSCHIntroduction") > 0 ){
```

Comment:
```suggestion
                  if(~_.indexOf(admintags, "ABSCHIntroduction"))){
```
instead of stringify